Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
```

Comment:
Honestly I don't know what `satisfy` is expecting, so I think returning a boolean should be fine. the find method in the `firefox_installation` block returns either the found path string or nil, so testing with `nil?` should be fine. And you're right, it should be `satisfy { !firefox_installation.nil? }`